lib/internal/test_runner/tap_stream.js

```diff
@@ -131,9 +131,18 @@ class TapStream extends Readable {
 
 // In certain places, # and \ need to be escaped as \# and \\.
 function tapEscape(input) {
-  return StringPrototypeReplaceAll(
-    StringPrototypeReplaceAll(input, '\\', '\\\\'), '#', '\\#'
-  );
+  [{ escapeChar: '\\', tappedEscape: '\\\\' },
```

Comment:
Even though the resulting code will not look as nice as this, I'm inclined to use a series of `StringPrototypeReplaceAll()` calls here instead.

Reply:
@cjihrig  like this... right?

```js
return StringPrototypeReplaceAll(StringPrototypeReplaceAll(
    StringPrototypeReplaceAll(StringPrototypeReplaceAll(
      StringPrototypeReplaceAll(StringPrototypeReplaceAll(
        StringPrototypeReplaceAll(StringPrototypeReplaceAll(input, '\\', '\\\\'), '#', '\\#'),
        '\b', '\\b'), '\f', '\\f'), '\t', '\\t'), '\n', '\\n'), '\r', '\\r'), '\v', '\\v');
```

Reply:
You could do it that way. You could also do something like this to maintain more readability:

```js
let result = StringPrototypeReplaceAll(...);
result = StringPrototypeReplaceAll(...);
result = StringPrototypeReplaceAll(...);
return result;
```

Reply:
Got it 👍🏽 
Thanks